src/components/navigation/NavigationItems.tsx

```diff
@@ -2,13 +2,31 @@ import React from 'react';
 
 import {
   BusinessCenter as BusinessCenterIcon,
+  CalendarTodayOutlined,
   Feedback as FeedbackIcon,
   GitHub as GithubIcon,
```

Comment:
If we wanna have outlined icons, we should change them all to be outlined

Reply:
good point, replaced in c1be1406441c7d554e05b14f660e9d69c8665206